cloudcopy/server/api/v0/schemas.py

```diff
@@ -0,0 +1,60 @@
+from typing import Optional, List
+from pydantic import BaseModel
```

Comment:
FastAPI schema, via Pydantic: structure of API input objects (e.g. request bodies) and output object (e.g. response bodies)